randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1045,15 +1066,20 @@ Extra - room_id: [25]
           All of the following:
               Gravity Suit
               Any of the following:
-                  Space Jump or Wall Jump (Intermediate)
+                  Space Jump
+                  # WJ with HJ: https://youtu.be/vku11Eov-m0
                   Hi-Jump and Wall Jump (Beginner)
+                  # WJ without HJ: https://youtu.be/TDwTWYNa_F8
+                  Wall Jump (Intermediate)
```

Comment:
see above. if you want to go the extra mile, you coud timestamp your videos where you stand on the ledge, but not really necessary.